src/komponenter/alleredeOpprettetTiltak/innholdsvisning/AlleredeOpprettetAvtale.tsx

```diff
@@ -18,26 +18,23 @@ const AlleredeOpprettetAvtale: React.FC<Props> = ({ alleredeRegistrertAvtale })
 
     return (
         <div className={cls.className}>
-            <div className={cls.element('info')}>
-                {sortBy(alleredeRegistrertAvtale, ['startDato', 'status']).map(
-                    (avtale: AlleredeRegistrertAvtale, index: number) => {
-                        return (
-                            <div key={index} className={cls.element('avtale')}>
-                                <div className={cls.element('periode-wrapper')}>
-                                    <RadInfo
-                                        label={'PERIODE:'}
-                                        info={visPeriodeForTiltak(avtale.startDato, avtale.sluttDato)}
-                                        infoNotBold={true}
-                                    />
-                                </div>
-                                <RadInfo label={'avtale nummer:'} info={avtale.avtaleNr} />
-                                <BodyShort size="small">Tiltakstype: {tiltakstypeTekst[avtale.tiltakstype]}</BodyShort>
-                                <AlleredeOpprettetAvtaleDetaljer innhold={avtale} />
-                            </div>
-                        );
-                    },
-                )}
-            </div>
+            {sortBy(alleredeRegistrertAvtale, ['startDato', 'status']).map(
+                (avtale: AlleredeRegistrertAvtale, index: number) => {
+                    return (
+                        <div key={index} className={cls.element('avtale')}>
```

Comment:
Kanskje bruke avtale id som key istedenfor index... 💀

Reply:
Enig, det er bedre praksis 👍 

Skal sies at det ikke vil være et problem i praksis for denne listen, ettersom indeks-baserte nøkler er problematiske når innhold endrer seg i komponenten.

Reply:
d4291535f748fda8471b4c2a3f0fb3e920f61fc5